### ISSUE TITLE
zoomToSpan inconsistence

### ISSUE THREAD

**Issue by JockeLindberg** [bryntum/support#10294]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=30559&p=157300#p157300)

Using the posters code, using zoomToSpan multiple times causes inconsistent zoomlevels.
This could be a user issue as well, but needs som investigation.

**Comment by ghulamghousdev** [comment]
+1 https://forum.bryntum.com/viewtopic.php?p=160212#p160212

**Comment by berrutti** [comment]
I can also reproduce. In our project, zooming to a very short span of dates (5 to 6) zooms in correctly, but then consecutive zoomToSpan with the same span size but different days (like 6 to 7, 9 to 10, 12 to 13, etc) returns different zoom levels, when they should be the same (just different days)

**Comment by ghulamghousdev** [comment]
I have used the code in this PR: https://github.com/bryntum/bryntum-suite/pull/12394 to check this issue. The issue is not reproducible. Once above mentioned PR is merged, we'll check again.

**Comment by ghulamghousdev** [comment]
We have fixed a similar issue in another ticket and I was not able to reproduce this one anymore with that fix, can you please recheck with version 7.0.2 and let us know if it works for you?

**Comment by ghulamghousdev** [comment]
Closing this ticket for now, will reopen it, if client reports that issue still persists.
